source/common/common/backoff_strategy.cc

```diff
@@ -0,0 +1,35 @@
+#include "common/common/backoff_strategy.h"
+
+namespace Envoy {
+
+ExponentialBackOffStrategy::ExponentialBackOffStrategy(const std::uint64_t initial_interval,
+                                                       const std::uint64_t max_interval,
```

Comment:
I think it would be good to remove the `std::` prefix on all the numeric types in this PR.